common/pupil/screening.ts

```diff
@@ -74,6 +84,11 @@ export async function updatePupilScreening(screener: Screener, pupilScreeningId:
             break;
 
         case PupilScreeningStatus.pending:
+            if (isFirstScreening) {
```

Comment:
No this is wrong, a screening will also be pending if it is just edited by a screener without taking an action.

Reply:
Cause "missing the screening" is not really a "state", a user can miss the screening multiple times. Also it does not really matter if the user does not show up at all or misses the screening.